testing/consul/APKBUILD

```diff
@@ -61,9 +37,13 @@ prepare() {
 
 build() {
 	cd "$_builddir"/$pkgname || return 1
-	export GOPATH="$srcdir"
-	go build -v -o bin/consul \
-		-ldflags "-X main.GitDescribe $pkgver" || return 1
+
+	mkdir -p "${_builddir}/src/github.com/hashicorp"
```

Comment:
This will result in `.../src/consul-0.6.4/src/github.com/hashicorp/consul/`. I found it was better to `mkdir -p $srcdir/github.com/hashicorp && ln -s $builddir $srcdir/github.com/hashicorp/consul`. I'm not sure what others think about that though. @clandmeter @ncopa @fcolista what do you think of re-using srcdir as the go src path?